components/Accordion/src/index.scss

```diff
@@ -52,6 +52,8 @@
 .denhaag-accordion__title {
   --utrecht-heading-5-margin-block-start: 0;
   --utrecht-heading-5-margin-block-end: 0;
+  --utrecht-heading-5-font-family: var(--denhaag-accordion-font-family);
+  --utrecht-heading-5-font-weight: 400;
```

Comment:
Apply var(--denhaag-typography-weight-regular);  (not sure if this is the 400 value)